### PR TITLE
Added functions to check etag of elements

### PR DESCRIPTION
**Pull request by SergioBertolinSG**
Improvements to play with etags.

### CODE REVIEW

**Comment by PVince81** [comment]
Adjust the comment then it's good :+1:

File: build/integration/features/bootstrap/WebDav.php [hunk 1]
```diff
@@ -16,6 +16,8 @@
 	private $usingOldDavPath = true;
 	/** @var ResponseInterface */
 	private $response;
+	/** @var array */
```
Comment (SergioBertolinSG) [inline build/integration/features/bootstrap/WebDav.php]: Actually it is a map with user as key an another map as value, which has path as key and etag as value.

Comment (SergioBertolinSG) [inline build/integration/features/bootstrap/WebDav.php]: Changed.

**Comment by PVince81** [comment]
:+1: please merge if you're done. (label still says "2 - Developing")

**Pull request merged**